### ISSUE TITLE
"More" buttons on support page are not keyboard accessible

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4040]
## Area or Component
https://stg.simplereport.gov/support/

## Quick Summary
The "more" buttons underneath "Account management", "Testing", and "Getting started" are not keyboard accessible - you can't tab to them.

This is likely because they're not `<button>` elements but rather `<div>` elements operating as a button.

## Accessibility Violation
Action cannot be performed by keyboard alone: https://cdc-devtools.dequecloud.com/rule-help/en/keyboard-inaccessible

## Action Requested
Change these to actually be `<button>` elements, or another clickable element that will expand.

## Screenshots (optional)
![Screen Shot 2022-07-21 at 6 31 16 PM](https://user-images.githubusercontent.com/80282552/180342317-923f3c90-8a3b-4089-8b63-fd98da266e7e.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1832428693&range=4:9

**Comment by emyl3** [comment]
Paired with @kenieh on this in regards to styling

**BEFORE**

<img width="340" alt="Screen Shot 2022-08-03 at 3 03 52 PM" src="https://user-images.githubusercontent.com/20211771/182698945-03e851d9-6cee-4a3b-8f91-e4a1e85084de.png">


**AFTER**
<img width="332" alt="Screen Shot 2022-08-03 at 3 03 41 PM" src="https://user-images.githubusercontent.com/20211771/182698991-a2fc4fdf-75c8-43a1-837d-433bcedf3e96.png">

"More" will be a `<button>` that has the `usa-button usa-button--unstyled` classes

The links on the page will be updated to have the `usa-link` class.

Will create a story to update across all links in the SimpleReport static site outside of the support page.
